### ISSUE TITLE
Apple/Google/Microsoft Pay

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1746]
Adding these payment methods could technically be part of #680, but I'm opening a separate issue for them so that everyone can express their opinions on whether we should support these branded protocols.

**Comment by trebmuh** [comment]
What would it allows from a donator and a receiver perspective ?

**Comment by Changaco** [comment]
@trebmuh My understanding is that these protocols allow paying with a card number that has previously been stored, so it's easier for the payer, and without actually revealing the card number to the merchant, so it's “safer”.

**Comment by LiamDawe** [comment]
Google Pay support would definitely be appreciated, it would open up a lot more people to be able to pledge easily.

**Comment by MrTeeXD** [comment]
I agree with the Google Pay option. Since most people don't have Paypal or a bank account, Google Pay would make it easier for all to Pledge.

**Comment by iMonZ** [comment]
Apple Pay and Google Pay are a must-have in the current day
